src/Apps/Auction/Routes/ConfirmBid/index.tsx

```diff
@@ -34,8 +37,12 @@ interface ConfirmBidProps {
   location: Location
 }
 
+const MAX_POLL_ATTEMPTS = 20
+
 export const ConfirmBidRoute: React.FC<ConfirmBidProps> = props => {
-  const { artwork } = props
+  let pollCount = 0
```

Comment:
minor question here whether there is a better way to do this using a react hook.